_pytest/main.py

```diff
@@ -403,7 +403,7 @@ def _repr_failure_py(self, excinfo, style=None):
             else:
                 style = "long"
 
-        return excinfo.getrepr(funcargs=True,
+        return excinfo.getrepr(funcargs=True, abspath=True,
```

Comment:
I agree if it changes the way paths are output.

IMHO ideally it'd not change the output at all and use relative paths, *except* if the cwd doesn't exist anymore (then show absolute paths). That'd be the nicer solution IMHO, but I haven't looked at the code.